docs/docs/getting-started/client.md

```diff
@@ -165,6 +165,8 @@ See repository [`README`](https://github.com/nextauthjs/react-query) for more de
 
 NextAuth.js provides a `getSession()` helper which should be called **client side only** to return the current active session.
 
+On the server side, **this is still available to use**, however, we recommend using `unstable_getServerSession` going forward. The idea behind this is to avoid an additional unnecessary `fetch` call on the server side. For more information, please check out [this issue](https://github.com/nextauthjs/next-auth/issues/1535).
```

Comment:
Let's mention the reason for being `unstable_`. (Because the API might change, not because it might cause issues. If there are, we will address them in bug fixes ASAP)